### ISSUE TITLE
Windows: no module named functl

### ISSUE THREAD

**Issue by nextdayy** [idlesign/keybind#4]
Python 3.9.1 (tags/v3.9.1:1e5d33e, Dec  7 2020, 17:08:21) [MSC v.1927 64 bit (AMD64)] on win32:

On this version: when the example is ran you recieve an error call;
![image](https://user-images.githubusercontent.com/79922345/109663907-8435da80-7b64-11eb-8a1c-da7056379773.png)

you need any more infomation?

**Comment by idlesign** [comment]
> you need any more infomation?

No, that's enough.

From README:

> Requires X11 (X Window System). For UNIX-like systems, e.g. Linux.

Link #3

**Comment by nextdayy** [comment]
what is the problem?

**Comment by nextdayy** [comment]
you saying this works on Linux only? You should make that more clear in the documentation.

**Comment by idlesign** [comment]
> you saying this works on Linux only?

Nope. The readme says that keybind works on X11 (X Window System), that's on various UNIX-like systems, for example Linux.

>  You should make that more clear in the documentation.

Your suggestions on documentation improvements are welcome.

**Comment by b1ek** [comment]
> The readme says that keybind works on X11 (X Window System), that's on various UNIX-like systems, for example Linux.

Does this means it depends on some another package on windows?

How to solve the damn problem?

That package is not available on pip, because its a default library, but only on linux<br/>
WIndows uses `win32api`, not `fcntl`
